AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -67,6 +91,29 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
         self.context = context
     }
     
+    func deleteSession(_ sessionUUID: SessionUUID) throws {
+        do {
+            try context.delete(context.existingSession(uuid: sessionUUID))
+        } catch {
+            Log.info("## Error when deleting session")
```

Comment:
let's log this error (it's available by default as `error` in the `catch` block)